tox.ini

```diff
@@ -31,7 +31,7 @@ commands =
     lib: pip install --editable .[dev]
     lib: make flake8
     lib: make pylint
-    lib: make test_codebase
+    lib: make test_lib
```

Comment:
is this essentially `test_self`? `lib` is more open to interpretation than `codebase`.